### PR TITLE
Create YNAB.json

### PR DESCRIPTION
**Pull request by george-dilthey**
Added george_dilthey.json

### CODE REVIEW

File: organizations/george_dilthey.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (pfrisella) [inline organizations/george_dilthey.json:1]: Which source belongs to this org? I suspect this is for YNAB but the org should be the org of YNAB. Note that the sources and orgs do not represent a connector, they simply define a data source.

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: organizations/george_dilthey.json (hunk 1 above)
Comment (george-dilthey) [reply organizations/george_dilthey.json:1]: Hi @pfrisella! I'm new to this, so thanks for bearing with me! I'm building out a connector using the YNAB API, but I'm not associated with YNAB at all, so I'm not 100% sure what the SOP should be here. I didn't want to create a new org saying that I represent YNAB, but I also wanted it to be clear that the data source is coming from YNAB. Let me know what you think and I can edit things accordingly.

Comment (pfrisella) [reply organizations/george_dilthey.json:1]: This repo just describes/identifies data sources, all of which is public information. It does not represent your connector or associate you with a company.

**Comment by pfrisella** [comment]
Closing due to inactivity.

**Comment by george-dilthey** [comment]
Hey Pete. Not sure why this got closed. I was waiting on a review from you for the latest version, and the source that was closed relied on this. I'm happy to submit again if need be, just let me know.

**Pull request closed without merging**